cmd/ctl/pkg/install/helm/settings.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helm
+
+import (
+	"github.com/spf13/pflag"
+
+	"helm.sh/helm/v3/pkg/cli"
+)
+
+func CopyCliFlags(flags *pflag.FlagSet, defaults map[string]string, cliEnvSettings *cli.EnvSettings) error {
```

Comment:
Please document the intended behaviour of this function. 
And consider renaming it, because I don't understand in what sense it *copies* flags?

It seems to merge the supplied flags with some overrides and then runs parse.